ts/masters/modules/stock.src.ts

```diff
@@ -22,8 +22,10 @@ import '../../Core/Navigator.js';
 import '../../Extensions/RangeSelector.js';
 import StockChart from '../../Core/Chart/StockChart.js';
 const G: AnyRecord = Highcharts;
+// Compositions
+if (!G.Scrollbar) {
+    Scrollbar.compose(G.Axis);
+}
 // Classes
 G.Scrollbar = Scrollbar;
 G.StockChart = G.stockChart = StockChart.stockChart;
-// Compositions
-Scrollbar.compose(G.Axis);
```

Comment:
The order should not be important for the composition here. (= compositions usually come last, after all namespace assignments) Was something failing?